.gitmodules

```diff
@@ -0,0 +1,16 @@
+[submodule "src/static/codeql/codeql"]
+	path = src/static/codeql/codeql
+	url = https://github.com/github/codeql.git
+	branch = rc/1.26
+	ignore = all
+[submodule "sut/openenclave"]
+	path = sut/openenclave
+	url = https://github.com/openenclave/openenclave.git
+	branch = master
+	ignore = all
+[submodule "sut/oeedger8r-cpp"]
```

Comment:
oeedger8r is also a submodule of openenclave. There is also the option of running codeql on that submodule instead of directly  on the repo.
Is there a benefit to running codeql directly on the oeedger8r repo?